.github/workflows/release.yaml

```diff
@@ -1,46 +1,47 @@
-name: release
+name: Release KDIS
 
 on:
-    workflow_run:
-        workflows:
-            - check
-        types:
-            - completed
+  push:
```

Comment:
This workflow must be run after the check workflow has completed successfully to avoid compilation errors, formatting, and linting issues. Having merely 'tags' run every time a matching tag is released without any check.

Reply:
Okay, one minor complaint about the way it is now is that the workflows list is pretty noisy with many skipped `release` runs, which leaves clutter in the Actions tab on GitHub. Perhaps there's a way for the `release` workflow to require a `check` workflow rerun, so that no one could do an unchecked release?

Reply:
I don't think we can hide it since GitHub shows all triggered workflows. Even though the release is triggered it's immediately skipped so it's not a big deal. If you find a way to skip it without running :)